src/_tests/fixtures/43136/derived.json

```diff
@@ -1,7 +1,8 @@
 {
   "type": "info",
-  "now": "2020-04-29T16:18:32Z",
+  "now": "2020-03-19T19:14:42.000Z",
```

Comment:
Had to update all the snapshots because the hand-entered dates didn’t agree with `stalenessInDays`.